DefaultSettings/getDefaultSettings.m

```diff
@@ -379,7 +477,20 @@
     S.subject.set_limit_torque_coefficients_selected_dofs = []; 
 end
 
+% ligament stiffness for all ligaments
+if ~isfield(S.subject,'stiffness_all_ligaments')
+    S.subject.stiffness_all_ligaments = 'ligamentGefen2002'; 
+end
+
+% ligament stiffness for selected ligaments
+if ~isfield(S.subject,'set_stiffness_selected_ligaments')
+    S.subject.set_stiffness_selected_ligaments = {'PlantarFascia','plantarFasciaNatali2010'};
+end
```

Comment:
Is this related to the updated bounds?

Reply:
these settings also exist on master, so not sure why it shows up as a change here